Source/WebCore/contentextensions/ContentExtensionParser.cpp

```diff
@@ -206,13 +206,18 @@ bool isValidCSSSelector(const String& selector)
 {
     ASSERT(isMainThread());
     ProcessWarming::initializeNames();
-    CSSParser parser(contentExtensionCSSParserContext());
+
+    // This explicitly does not use the CSSParserContext created in contentExtensionCSSParserContext because
+    // we want to use quirks mode in parsing, but automatic mode when actually applying the content blocker styles.
```

Comment:
Could add a FIXME here to figure out how to clean this up.

Reply:
Added:

```
// FIXME: rdar://105733691 (Parse/apply content blocker style sheets in both standards and quirks mode lazily).
```